esmvaltool/recipes/recipe_Thermodyn_diagtool.yml

```
@@ -0,0 +1,245 @@
+    #- {dataset: MPI-ESM-MR, project: CMIP5, exp: rcp85, ensemble: r1i1p1, start_year: 2070, end_year: 2099}
+    #- {dataset: MPI-ESM-MR, project: CMIP5, exp: rcp85, ensemble: r1i1p1, start_year: 2070, end_year: 2073}
+    #- {dataset: MPI-ESM-P, project: CMIP5, exp: piControl, ensemble: r1i1p1, start_year: 2986, end_year: 3005}
+    #- {dataset: MPI-ESM-P, project: CMIP5, exp: piControl, ensemble: r1i1p1, start_year: 2986, end_year: 2991}
+    #- {dataset: MPI-ESM-P, project: CMIP5, exp: historical, ensemble: r1i1p1, start_year: 1980, end_year: 1999}
+preprocessors:
+  prep0: {}
+
+diagnostics:
+  Thermodyn_Diag:
+    description: Thermodynamics diagnostics
+    themes: phys
+    realms: atmos
+    variables:
+       hfls:
+         preprocessor: prep0
+         mip: Amon
+         field: T2Ms
+       hfss:
+         preprocessor: prep0
+         mip: Amon
+         field: T2Ms
+       pr:
+         preprocessor: prep0
+         mip: Amon
+         field: T2Ms
+       ps:
+         preprocessor: prep0
+         mip: Amon
+         field: T2Ms
+       prsn:
+         preprocessor: prep0
+         mip: Amon
+         field: T2Ms
+       rlds:
+         preprocessor: prep0
+         mip: Amon
+         field: T2Ms
+       rlus:
+         preprocessor: prep0
+         mip: Amon
+         field: T2Ms
+       rlut:
+         preprocessor: prep0
+         mip: Amon
+         field: T2Ms
+       rsds:
+         preprocessor: prep0
+         mip: Amon
+         field: T2Ms
+       rsus:
+         preprocessor: prep0
+         mip: Amon
+         field: T2Ms
+       rsdt:
+         preprocessor: prep0
+         mip: Amon
+         field: T2Ms
+       rsut:
+         preprocessor: prep0
+         mip: Amon
+         field: T2Ms
+       #sftlf:
+     # #  preprocessor: prep0
+     # #  mip: fx
+     # #  field: F2Ms
+       ts:
+         preprocessor: prep0
+         mip: Amon
+         field: T2Ms
+       #uas:
+      #   preprocessor: prep0
+      #   mip: Amon
+      #   field: T2Ms
+      # vas:
+      #   preprocessor: prep0
+      #   mip: Amon
+      #   field: T2Ms
+       hus:
+         preprocessor: prep0
+         mip: Amon
+         field: T3M
+       tas:
+         preprocessor: prep0
+         mip: day
+         field: T2Ds
+       uas:
+         preprocessor: prep0
+         mip: day
+         field: T2Ds
+       vas:
+         preprocessor: prep0
+         mip: day
+         field: T2Ds
+       ta:
+         preprocessor: prep0
+         fx_files: [sftlf]
+         mip: day
+         field: T3D
+       ua:
+         preprocessor: prep0
+         mip: day
+         field: T3D
+       va:
+         preprocessor: prep0
+         mip: day
+         field: T3D
+       wap:
+         preprocessor: prep0
+         mip: day
+         field: T3D
+    scripts:
+      Thermodyn_Diag:
+        script: Thermodyn_diagtool/thermodyn_diagnostics.py
+        eb: y
```

The correct yaml syntax is:
```suggestion
        eb: true
```
Please make use of `yamllint` to check your yaml files.

---

I tested the recipe with yamllint, and now it is all ok. I changed 'y' to 'true'